Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3125,7 +3125,11 @@ - (void)_presentLockdownMode
     if (!lockdownModeWarningNeeded)
         return;
 
+#if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST)
```

Comment:
No need for `PLATFORM(IOS_FAMILY)` since the entire file is guarded by that.

Would re-write as:

```
#if PLATFORM(MACCATALYST)
#else
#endif
```

Reply:
Will do!